contracts/TalentLayerID.sol

```diff
@@ -106,16 +109,7 @@ contract TalentLayerID is ERC2771RecipientUpgradeable, ERC721Upgradeable, UUPSUp
      * @return uint256 the id of the NFT
      */
     function walletOfOwner(address _owner) public view returns (uint256) {
```

Comment:
now that we got the mapping, I think we can just pass it as public and use the automatic getter.
Can you check in the same time if it improves the gas usage ?

Reply:
We save 34gas using the public automatic getter.
& we can remove the "walletOfOwner" function